mizar/common/cidr.py

```diff
@@ -31,7 +33,7 @@ def __init__(self, prefixlen, ip):
         self.ip = ip
         self.ipnet = ipaddress.ip_network(
             "{}/{}".format(self.ip, self.prefixlen))
-        self.subnets = self.ipnet.subnets(new_prefix=30)
+        self.subnets = self.ipnet.subnets(new_prefix=32)
```

Comment:
Can we define this 32 as a constant. I'm just curious why we picked that number. This change seems to work.
```
root@fw0000358:~/go/src/k8s.io/varktos# python3
Python 3.7.5 (default, Dec  9 2021, 17:04:37) 
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import ipaddress
>>> ipaddress.ip_network("20.0.0.0/24")
IPv4Network('20.0.0.0/24')
>>> ipnw = ipaddress.ip_network("20.0.0.0/24")
>>> list(ipnw)
[IPv4Address('20.0.0.0'), IPv4Address('20.0.0.1'), IPv4Address('20.0.0.2'), IPv4Address('20.0.0.3'), IPv4Address('20.0.0.4'), IPv4Address('20.0.0.5'), IPv4Address('20.0.0.6'), IPv4Address('20.0.0.7'), IPv4Address('20.0.0.8'), IPv4Address('20.0.0.9'), IPv4Address('20.0.0.10'), IPv4Address('20.0.0.11'), IPv4Address('20.0.0.12'), IPv4Address('20.0.0.13'), IPv4Address('20.0.0.14'), IPv4Address('20.0.0.15'), IPv4Address('20.0.0.16'), IPv4Address('20.0.0.17'), IPv4Address('20.0.0.18'), IPv4Address('20.0.0.19'), IPv4Address('20.0.0.20'), IPv4Address('20.0.0.21'), IPv4Address('20.0.0.22'), IPv4Address('20.0.0.23'), IPv4Address('20.0.0.24'), IPv4Address('20.0.0.25'), IPv4Address('20.0.0.26'), IPv4Address('20.0.0.27'), IPv4Address('20.0.0.28'), IPv4Address('20.0.0.29'), IPv4Address('20.0.0.30'), IPv4Address('20.0.0.31'), IPv4Address('20.0.0.32'), IPv4Address('20.0.0.33'), IPv4Address('20.0.0.34'), IPv4Address('20.0.0.35'), IPv4Address('20.0.0.36'), IPv4Address('20.0.0.37'), IPv4Address('20.0.0.38'), IPv4Address('20.0.0.39'), IPv4Address('20.0.0.40'), IPv4Address('20.0.0.41'), IPv4Address('20.0.0.42'), IPv4Address('20.0.0.43'), IPv4Address('20.0.0.44'), IPv4Address('20.0.0.45'), IPv4Address('20.0.0.46'), IPv4Address('20.0.0.47'), IPv4Address('20.0.0.48'), IPv4Address('20.0.0.49'), IPv4Address('20.0.0.50'), IPv4Address('20.0.0.51'), IPv4Address('20.0.0.52'), IPv4Address('20.0.0.53'), IPv4Address('20.0.0.54'), IPv4Address('20.0.0.55'), IPv4Address('20.0.0.56'), IPv4Address('20.0.0.57'), IPv4Address('20.0.0.58'), IPv4Address('20.0.0.59'), IPv4Address('20.0.0.60'), IPv4Address('20.0.0.61'), IPv4Address('20.0.0.62'), IPv4Address('20.0.0.63'), IPv4Address('20.0.0.64'), IPv4Address('20.0.0.65'), IPv4Address('20.0.0.66'), IPv4Address('20.0.0.67'), IPv4Address('20.0.0.68'), IPv4Address('20.0.0.69'), IPv4Address('20.0.0.70'), IPv4Address('20.0.0.71'), IPv4Address('20.0.0.72'), IPv4Address('20.0.0.73'), IPv4Address('20.0.0.74'), IPv4Address('20.0.0.75'), IPv4Address('20.0.0.76'), IPv4Address('20.0.0.77'), IPv4Address('20.0.0.78'), IPv4Address('20.0.0.79'), IPv4Address('20.0.0.80'), IPv4Address('20.0.0.81'), IPv4Address('20.0.0.82'), IPv4Address('20.0.0.83'), IPv4Address('20.0.0.84'), IPv4Address('20.0.0.85'), IPv4Address('20.0.0.86'), IPv4Address('20.0.0.87'), IPv4Address('20.0.0.88'), IPv4Address('20.0.0.89'), IPv4Address('20.0.0.90'), IPv4Address('20.0.0.91'), IPv4Address('20.0.0.92'), IPv4Address('20.0.0.93'), IPv4Address('20.0.0.94'), IPv4Address('20.0.0.95'), IPv4Address('20.0.0.96'), IPv4Address('20.0.0.97'), IPv4Address('20.0.0.98'), IPv4Address('20.0.0.99'), IPv4Address('20.0.0.100'), IPv4Address('20.0.0.101'), IPv4Address('20.0.0.102'), IPv4Address('20.0.0.103'), IPv4Address('20.0.0.104'), IPv4Address('20.0.0.105'), IPv4Address('20.0.0.106'), IPv4Address('20.0.0.107'), IPv4Address('20.0.0.108'), IPv4Address('20.0.0.109'), IPv4Address('20.0.0.110'), IPv4Address('20.0.0.111'), IPv4Address('20.0.0.112'), IPv4Address('20.0.0.113'), IPv4Address('20.0.0.114'), IPv4Address('20.0.0.115'), IPv4Address('20.0.0.116'), IPv4Address('20.0.0.117'), IPv4Address('20.0.0.118'), IPv4Address('20.0.0.119'), IPv4Address('20.0.0.120'), IPv4Address('20.0.0.121'), IPv4Address('20.0.0.122'), IPv4Address('20.0.0.123'), IPv4Address('20.0.0.124'), IPv4Address('20.0.0.125'), IPv4Address('20.0.0.126'), IPv4Address('20.0.0.127'), IPv4Address('20.0.0.128'), IPv4Address('20.0.0.129'), IPv4Address('20.0.0.130'), IPv4Address('20.0.0.131'), IPv4Address('20.0.0.132'), IPv4Address('20.0.0.133'), IPv4Address('20.0.0.134'), IPv4Address('20.0.0.135'), IPv4Address('20.0.0.136'), IPv4Address('20.0.0.137'), IPv4Address('20.0.0.138'), IPv4Address('20.0.0.139'), IPv4Address('20.0.0.140'), IPv4Address('20.0.0.141'), IPv4Address('20.0.0.142'), IPv4Address('20.0.0.143'), IPv4Address('20.0.0.144'), IPv4Address('20.0.0.145'), IPv4Address('20.0.0.146'), IPv4Address('20.0.0.147'), IPv4Address('20.0.0.148'), IPv4Address('20.0.0.149'), IPv4Address('20.0.0.150'), IPv4Address('20.0.0.151'), IPv4Address('20.0.0.152'), IPv4Address('20.0.0.153'), IPv4Address('20.0.0.154'), IPv4Address('20.0.0.155'), IPv4Address('20.0.0.156'), IPv4Address('20.0.0.157'), IPv4Address('20.0.0.158'), IPv4Address('20.0.0.159'), IPv4Address('20.0.0.160'), IPv4Address('20.0.0.161'), IPv4Address('20.0.0.162'), IPv4Address('20.0.0.163'), IPv4Address('20.0.0.164'), IPv4Address('20.0.0.165'), IPv4Address('20.0.0.166'), IPv4Address('20.0.0.167'), IPv4Address('20.0.0.168'), IPv4Address('20.0.0.169'), IPv4Address('20.0.0.170'), IPv4Address('20.0.0.171'), IPv4Address('20.0.0.172'), IPv4Address('20.0.0.173'), IPv4Address('20.0.0.174'), IPv4Address('20.0.0.175'), IPv4Address('20.0.0.176'), IPv4Address('20.0.0.177'), IPv4Address('20.0.0.178'), IPv4Address('20.0.0.179'), IPv4Address('20.0.0.180'), IPv4Address('20.0.0.181'), IPv4Address('20.0.0.182'), IPv4Address('20.0.0.183'), IPv4Address('20.0.0.184'), IPv4Address('20.0.0.185'), IPv4Address('20.0.0.186'), IPv4Address('20.0.0.187'), IPv4Address('20.0.0.188'), IPv4Address('20.0.0.189'), IPv4Address('20.0.0.190'), IPv4Address('20.0.0.191'), IPv4Address('20.0.0.192'), IPv4Address('20.0.0.193'), IPv4Address('20.0.0.194'), IPv4Address('20.0.0.195'), IPv4Address('20.0.0.196'), IPv4Address('20.0.0.197'), IPv4Address('20.0.0.198'), IPv4Address('20.0.0.199'), IPv4Address('20.0.0.200'), IPv4Address('20.0.0.201'), IPv4Address('20.0.0.202'), IPv4Address('20.0.0.203'), IPv4Address('20.0.0.204'), IPv4Address('20.0.0.205'), IPv4Address('20.0.0.206'), IPv4Address('20.0.0.207'), IPv4Address('20.0.0.208'), IPv4Address('20.0.0.209'), IPv4Address('20.0.0.210'), IPv4Address('20.0.0.211'), IPv4Address('20.0.0.212'), IPv4Address('20.0.0.213'), IPv4Address('20.0.0.214'), IPv4Address('20.0.0.215'), IPv4Address('20.0.0.216'), IPv4Address('20.0.0.217'), IPv4Address('20.0.0.218'), IPv4Address('20.0.0.219'), IPv4Address('20.0.0.220'), IPv4Address('20.0.0.221'), IPv4Address('20.0.0.222'), IPv4Address('20.0.0.223'), IPv4Address('20.0.0.224'), IPv4Address('20.0.0.225'), IPv4Address('20.0.0.226'), IPv4Address('20.0.0.227'), IPv4Address('20.0.0.228'), IPv4Address('20.0.0.229'), IPv4Address('20.0.0.230'), IPv4Address('20.0.0.231'), IPv4Address('20.0.0.232'), IPv4Address('20.0.0.233'), IPv4Address('20.0.0.234'), IPv4Address('20.0.0.235'), IPv4Address('20.0.0.236'), IPv4Address('20.0.0.237'), IPv4Address('20.0.0.238'), IPv4Address('20.0.0.239'), IPv4Address('20.0.0.240'), IPv4Address('20.0.0.241'), IPv4Address('20.0.0.242'), IPv4Address('20.0.0.243'), IPv4Address('20.0.0.244'), IPv4Address('20.0.0.245'), IPv4Address('20.0.0.246'), IPv4Address('20.0.0.247'), IPv4Address('20.0.0.248'), IPv4Address('20.0.0.249'), IPv4Address('20.0.0.250'), IPv4Address('20.0.0.251'), IPv4Address('20.0.0.252'), IPv4Address('20.0.0.253'), IPv4Address('20.0.0.254'), IPv4Address('20.0.0.255')]
>>> snw = ipnw.subnets(new_prefix=32)
>>> list(snw)
[IPv4Network('20.0.0.0/32'), IPv4Network('20.0.0.1/32'), IPv4Network('20.0.0.2/32'), IPv4Network('20.0.0.3/32'), IPv4Network('20.0.0.4/32'), IPv4Network('20.0.0.5/32'), IPv4Network('20.0.0.6/32'), IPv4Network('20.0.0.7/32'), IPv4Network('20.0.0.8/32'), IPv4Network('20.0.0.9/32'), IPv4Network('20.0.0.10/32'), IPv4Network('20.0.0.11/32'), IPv4Network('20.0.0.12/32'), IPv4Network('20.0.0.13/32'), IPv4Network('20.0.0.14/32'), IPv4Network('20.0.0.15/32'), IPv4Network('20.0.0.16/32'), IPv4Network('20.0.0.17/32'), IPv4Network('20.0.0.18/32'), IPv4Network('20.0.0.19/32'), IPv4Network('20.0.0.20/32'), IPv4Network('20.0.0.21/32'), IPv4Network('20.0.0.22/32'), IPv4Network('20.0.0.23/32'), IPv4Network('20.0.0.24/32'), IPv4Network('20.0.0.25/32'), IPv4Network('20.0.0.26/32'), IPv4Network('20.0.0.27/32'), IPv4Network('20.0.0.28/32'), IPv4Network('20.0.0.29/32'), IPv4Network('20.0.0.30/32'), IPv4Network('20.0.0.31/32'), IPv4Network('20.0.0.32/32'), IPv4Network('20.0.0.33/32'), IPv4Network('20.0.0.34/32'), IPv4Network('20.0.0.35/32'), IPv4Network('20.0.0.36/32'), IPv4Network('20.0.0.37/32'), IPv4Network('20.0.0.38/32'), IPv4Network('20.0.0.39/32'), IPv4Network('20.0.0.40/32'), IPv4Network('20.0.0.41/32'), IPv4Network('20.0.0.42/32'), IPv4Network('20.0.0.43/32'), IPv4Network('20.0.0.44/32'), IPv4Network('20.0.0.45/32'), IPv4Network('20.0.0.46/32'), IPv4Network('20.0.0.47/32'), IPv4Network('20.0.0.48/32'), IPv4Network('20.0.0.49/32'), IPv4Network('20.0.0.50/32'), IPv4Network('20.0.0.51/32'), IPv4Network('20.0.0.52/32'), IPv4Network('20.0.0.53/32'), IPv4Network('20.0.0.54/32'), IPv4Network('20.0.0.55/32'), IPv4Network('20.0.0.56/32'), IPv4Network('20.0.0.57/32'), IPv4Network('20.0.0.58/32'), IPv4Network('20.0.0.59/32'), IPv4Network('20.0.0.60/32'), IPv4Network('20.0.0.61/32'), IPv4Network('20.0.0.62/32'), IPv4Network('20.0.0.63/32'), IPv4Network('20.0.0.64/32'), IPv4Network('20.0.0.65/32'), IPv4Network('20.0.0.66/32'), IPv4Network('20.0.0.67/32'), IPv4Network('20.0.0.68/32'), IPv4Network('20.0.0.69/32'), IPv4Network('20.0.0.70/32'), IPv4Network('20.0.0.71/32'), IPv4Network('20.0.0.72/32'), IPv4Network('20.0.0.73/32'), IPv4Network('20.0.0.74/32'), IPv4Network('20.0.0.75/32'), IPv4Network('20.0.0.76/32'), IPv4Network('20.0.0.77/32'), IPv4Network('20.0.0.78/32'), IPv4Network('20.0.0.79/32'), IPv4Network('20.0.0.80/32'), IPv4Network('20.0.0.81/32'), IPv4Network('20.0.0.82/32'), IPv4Network('20.0.0.83/32'), IPv4Network('20.0.0.84/32'), IPv4Network('20.0.0.85/32'), IPv4Network('20.0.0.86/32'), IPv4Network('20.0.0.87/32'), IPv4Network('20.0.0.88/32'), IPv4Network('20.0.0.89/32'), IPv4Network('20.0.0.90/32'), IPv4Network('20.0.0.91/32'), IPv4Network('20.0.0.92/32'), IPv4Network('20.0.0.93/32'), IPv4Network('20.0.0.94/32'), IPv4Network('20.0.0.95/32'), IPv4Network('20.0.0.96/32'), IPv4Network('20.0.0.97/32'), IPv4Network('20.0.0.98/32'), IPv4Network('20.0.0.99/32'), IPv4Network('20.0.0.100/32'), IPv4Network('20.0.0.101/32'), IPv4Network('20.0.0.102/32'), IPv4Network('20.0.0.103/32'), IPv4Network('20.0.0.104/32'), IPv4Network('20.0.0.105/32'), IPv4Network('20.0.0.106/32'), IPv4Network('20.0.0.107/32'), IPv4Network('20.0.0.108/32'), IPv4Network('20.0.0.109/32'), IPv4Network('20.0.0.110/32'), IPv4Network('20.0.0.111/32'), IPv4Network('20.0.0.112/32'), IPv4Network('20.0.0.113/32'), IPv4Network('20.0.0.114/32'), IPv4Network('20.0.0.115/32'), IPv4Network('20.0.0.116/32'), IPv4Network('20.0.0.117/32'), IPv4Network('20.0.0.118/32'), IPv4Network('20.0.0.119/32'), IPv4Network('20.0.0.120/32'), IPv4Network('20.0.0.121/32'), IPv4Network('20.0.0.122/32'), IPv4Network('20.0.0.123/32'), IPv4Network('20.0.0.124/32'), IPv4Network('20.0.0.125/32'), IPv4Network('20.0.0.126/32'), IPv4Network('20.0.0.127/32'), IPv4Network('20.0.0.128/32'), IPv4Network('20.0.0.129/32'), IPv4Network('20.0.0.130/32'), IPv4Network('20.0.0.131/32'), IPv4Network('20.0.0.132/32'), IPv4Network('20.0.0.133/32'), IPv4Network('20.0.0.134/32'), IPv4Network('20.0.0.135/32'), IPv4Network('20.0.0.136/32'), IPv4Network('20.0.0.137/32'), IPv4Network('20.0.0.138/32'), IPv4Network('20.0.0.139/32'), IPv4Network('20.0.0.140/32'), IPv4Network('20.0.0.141/32'), IPv4Network('20.0.0.142/32'), IPv4Network('20.0.0.143/32'), IPv4Network('20.0.0.144/32'), IPv4Network('20.0.0.145/32'), IPv4Network('20.0.0.146/32'), IPv4Network('20.0.0.147/32'), IPv4Network('20.0.0.148/32'), IPv4Network('20.0.0.149/32'), IPv4Network('20.0.0.150/32'), IPv4Network('20.0.0.151/32'), IPv4Network('20.0.0.152/32'), IPv4Network('20.0.0.153/32'), IPv4Network('20.0.0.154/32'), IPv4Network('20.0.0.155/32'), IPv4Network('20.0.0.156/32'), IPv4Network('20.0.0.157/32'), IPv4Network('20.0.0.158/32'), IPv4Network('20.0.0.159/32'), IPv4Network('20.0.0.160/32'), IPv4Network('20.0.0.161/32'), IPv4Network('20.0.0.162/32'), IPv4Network('20.0.0.163/32'), IPv4Network('20.0.0.164/32'), IPv4Network('20.0.0.165/32'), IPv4Network('20.0.0.166/32'), IPv4Network('20.0.0.167/32'), IPv4Network('20.0.0.168/32'), IPv4Network('20.0.0.169/32'), IPv4Network('20.0.0.170/32'), IPv4Network('20.0.0.171/32'), IPv4Network('20.0.0.172/32'), IPv4Network('20.0.0.173/32'), IPv4Network('20.0.0.174/32'), IPv4Network('20.0.0.175/32'), IPv4Network('20.0.0.176/32'), IPv4Network('20.0.0.177/32'), IPv4Network('20.0.0.178/32'), IPv4Network('20.0.0.179/32'), IPv4Network('20.0.0.180/32'), IPv4Network('20.0.0.181/32'), IPv4Network('20.0.0.182/32'), IPv4Network('20.0.0.183/32'), IPv4Network('20.0.0.184/32'), IPv4Network('20.0.0.185/32'), IPv4Network('20.0.0.186/32'), IPv4Network('20.0.0.187/32'), IPv4Network('20.0.0.188/32'), IPv4Network('20.0.0.189/32'), IPv4Network('20.0.0.190/32'), IPv4Network('20.0.0.191/32'), IPv4Network('20.0.0.192/32'), IPv4Network('20.0.0.193/32'), IPv4Network('20.0.0.194/32'), IPv4Network('20.0.0.195/32'), IPv4Network('20.0.0.196/32'), IPv4Network('20.0.0.197/32'), IPv4Network('20.0.0.198/32'), IPv4Network('20.0.0.199/32'), IPv4Network('20.0.0.200/32'), IPv4Network('20.0.0.201/32'), IPv4Network('20.0.0.202/32'), IPv4Network('20.0.0.203/32'), IPv4Network('20.0.0.204/32'), IPv4Network('20.0.0.205/32'), IPv4Network('20.0.0.206/32'), IPv4Network('20.0.0.207/32'), IPv4Network('20.0.0.208/32'), IPv4Network('20.0.0.209/32'), IPv4Network('20.0.0.210/32'), IPv4Network('20.0.0.211/32'), IPv4Network('20.0.0.212/32'), IPv4Network('20.0.0.213/32'), IPv4Network('20.0.0.214/32'), IPv4Network('20.0.0.215/32'), IPv4Network('20.0.0.216/32'), IPv4Network('20.0.0.217/32'), IPv4Network('20.0.0.218/32'), IPv4Network('20.0.0.219/32'), IPv4Network('20.0.0.220/32'), IPv4Network('20.0.0.221/32'), IPv4Network('20.0.0.222/32'), IPv4Network('20.0.0.223/32'), IPv4Network('20.0.0.224/32'), IPv4Network('20.0.0.225/32'), IPv4Network('20.0.0.226/32'), IPv4Network('20.0.0.227/32'), IPv4Network('20.0.0.228/32'), IPv4Network('20.0.0.229/32'), IPv4Network('20.0.0.230/32'), IPv4Network('20.0.0.231/32'), IPv4Network('20.0.0.232/32'), IPv4Network('20.0.0.233/32'), IPv4Network('20.0.0.234/32'), IPv4Network('20.0.0.235/32'), IPv4Network('20.0.0.236/32'), IPv4Network('20.0.0.237/32'), IPv4Network('20.0.0.238/32'), IPv4Network('20.0.0.239/32'), IPv4Network('20.0.0.240/32'), IPv4Network('20.0.0.241/32'), IPv4Network('20.0.0.242/32'), IPv4Network('20.0.0.243/32'), IPv4Network('20.0.0.244/32'), IPv4Network('20.0.0.245/32'), IPv4Network('20.0.0.246/32'), IPv4Network('20.0.0.247/32'), IPv4Network('20.0.0.248/32'), IPv4Network('20.0.0.249/32'), IPv4Network('20.0.0.250/32'), IPv4Network('20.0.0.251/32'), IPv4Network('20.0.0.252/32'), IPv4Network('20.0.0.253/32'), IPv4Network('20.0.0.254/32'), IPv4Network('20.0.0.255/32')]
>>> 
```

Reply:
Added a constant for 32.